cmd/url.go

```diff
@@ -179,16 +181,24 @@ var urlListCmd = &cobra.Command{
 			fmt.Printf("No URLs found for component %v in application %v\n", componentName, app)
 		} else {
 			fmt.Printf("Found the following URLs for component %v in application %v:\n", componentName, app)
+
+			tabWriterURL := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+
+			//create headers
+			fmt.Fprintln(tabWriterURL, "NAME", "\t", "URL", "\t", "PORT")
+
 			for _, u := range urls {
-				fmt.Printf("%v - %v\n", u.Name, url.GetUrlString(u))
+				fmt.Fprintln(tabWriterURL, u.Name, "\t", url.GetUrlString(u), "\t", u.Port.IntVal)
 			}
+			tabWriterURL.Flush()
 		}
 	},
 }
 
 func init() {
 	urlCreateCmd.Flags().StringVarP(&urlApplication, "application", "a", "", "create url for application")
 	urlCreateCmd.Flags().StringVarP(&urlComponent, "component", "c", "", "create url for component")
+	urlCreateCmd.Flags().IntVarP(&urlPort, "port", "", -1, "port number for the url of the component")
```

Comment:
Help message should explain that this flag is required only if component exposes more than one port. Otherwise, a component port will be used.